src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
```

Comment:
It makes sense to define 100 as constant.
Same note for 50 inside `Carnivore.bite()`.